Index.html

```diff
@@ -0,0 +1,189 @@
+                <p class="title-article">User Experience</p>
+                <p class="paragraph-article">In order to build successful products you
+                    have to understand the user. We create
+                    user personas, wireframes and prototypes
+                    to test with users. This allows us to gather
+                    feedback which helps to shape the product
+                    and can quickly eliminate potential risks –
+                    saving you time and money.</p>
+
+            </article>
+            <article class="article-settings-section-3">
+                <img class="article-section-3-img" src="assets/icon_graphic-design.svg" alt="Icon Graphic Design"/>
+                <p class="title-article">Logo Design + Branding</p>
+                <p class="paragraph-article">Nike, Apple, Coca-Cola, Starbucks. What do
+                    all of these brands have in common? Easily
+                    recognizable logos and brand identity. The
+                    best brands stand out from the crowd by
+                    having a clear and unique point of view. We
+                    will work with you to understand your
+                    vision and help create a unique brand
+                    identity and style guide for your company.</p>
+
+            </article>
+            <article class="article-settings-section-3">
+                <img class="article-section-3-img" src="assets/icon_seo.svg" alt="SEO"/>
+                <p class="title-article">SEO</p>
+                <p class="paragraph-article">Do you want your site to rank higher in
+                    search results? Google is constantly
+                    changing their algorithms for how they
+                    rank pages. Don’t worry about constantly
+                    playing catch-up. We will make sure your
+                    website it optimized, providing you the best
+                    opportunities for organic search exposure.</p>
+
+            </article>
+            <article class="article-settings-section-3">
+                <img class="article-section-3-img" src="assets/icon_copywriting.svg" alt="Icon Copywriting"/>
+                <p class="title-article">Copy Writing</p>
+                <p class="paragraph-article">Brand messaging needs to be clear, direct
+                    and easy to understand to be effective. A
+                    unique tone of voice will help you stand out
+                    from the crowd and build brand loyalty. We
+                    work with you to establish your brand bible
+                    and ensure your messaging is focused and
+                    targeted.</p>
+
+            </article>
+        </div>
+
+    </section>
+
+    <section class="section-4 outer-container">
+        <div class="form-section-4 inner-container">
+            <form action="">
+                <h5>Let's work together</h5>
```

Comment:
Deze titel hoort niet in het formulier thuis, het is immers geen invoerveld! Dus we stoppen alleen de labels, inputs en buttons binnen de form-tag!